.gitignore

```diff
@@ -46,4 +46,7 @@ env.bak/
 .packages/
 Pipfile
 Pipfile.lock
-coverage.json
\ No newline at end of file
+coverage.json
+
+# ignore .idea
+*.idea
```

Comment:
i think it should just be `.idea` instead of `*.idea` PyCharm creates a top-level directory named `.idea` so this should exclude all files.

If you want to put a comment then you can say `ignore IntelliJ’s project settings files`

Reply:
okay please remove this since @szuananwar has added this in https://github.com/buildtesters/buildtest/pull/1497